### PR TITLE
Config & table row component changes

### PR DESCRIPTION
**Pull request by Patryk-Komar**

### CODE REVIEW

File: src/components/TableRow.tsx [hunk 1]
```diff
@@ -183,14 +183,14 @@ class TableRow extends React.Component<IProps, IState> {
                 <TableCell>
                     {this.props.coins.toFixed(6)} {firstCurrency}
                 </TableCell>
-                <TableCell>
+                <StyledHighlightedProfitCell profit={orderProfitValue}>
                     { orderProfitValue !== 0
                         ? orderProfitValue > 0
```
Comment (TomaszChmielPagepro) [inline src/components/TableRow.tsx]:
```
? `${orderProfitValue.toFixed(8)} ${secondCurrency} ${orderProfitValue > 0 ? '(PROFIT)' : '(LOSS)'}`

File: src/config/config.ts [hunk 2]
```diff
@@ -10,8 +10,8 @@ const config: any = {
 }
 
 const apiConfig = {
-    config: '/api/config',
-    history: '/api/history'
+    config: `http://${envWebsocketDomain}/api/config`,
```
Comment (TomaszChmielPagepro) [inline src/config/config.ts]: This change is not needed, please revert it

Comment (klojniewski) [reply src/config/config.ts]: @TomaszChmielPagepro are you sure? In my opinion this is needed.

Comment (TomaszChmielPagepro) [reply src/config/config.ts]: https://github.com/Pagepro/arbitrage-frontend/blob/master/package.json#L32

We can change `target` fields here instead

File: src/config/config.ts [hunk 3]
```diff
@@ -10,8 +10,8 @@ const config: any = {
 }
 
 const apiConfig = {
-    config: '/api/config',
-    history: '/api/history'
+    config: `/api/config`,
```
Comment (TomaszChmielPagepro) [inline src/config/config.ts]: change ` to '

**Pull request merged**